### PR TITLE
ACC-111 fix EXO_HOST_PORT undefined & ensure docker container removal

### PR DESCRIPTION
**Pull request by madhifallah**
I found that EXO_HOST_PORT is wrongly calculated and also not set later for onlyoffice.documentserver.host
I needed to to use DEPLOYMENT_ONLYOFFICE_HTTP_PORT, and declare EXO_HOST_PORT  in setenv-local.sh & standalone-local.conf

Another fix comes here to ensure deleting docker container before attempting to delete its associated volumes.

### CODE REVIEW

File: _functions.sh [hunk 1]
```diff
@@ -1143,10 +1143,10 @@ do_deploy() {
 
   if [ -z ${DEPLOYMENT_APACHE_VHOST_ALIAS} ]; then
     env_var "DEPLOYMENT_URL" $(do_build_url "http" "${DEPLOYMENT_EXT_HOST}" "${DEPLOYMENT_EXT_PORT}" "")
-    env_var "EXO_HOST_PORT" "${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_EXT_PORT}"
+    env_var "EXO_HOST_PORT" "${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```
Comment (vsellier) [inline _functions.sh]: You should rename the variable to ONLY_OFFICE_HOST_PORT IMO

Comment (madhifallah) [reply _functions.sh]: Var renamed

File: etc/adt/config.template [hunk 2]
```diff
@@ -57,7 +57,7 @@ DEPLOYMENT_ES_IMAGE=${DEPLOYMENT_ES_IMAGE}
 DEPLOYMENT_ES_IMAGE_VERSION=${DEPLOYMENT_ES_IMAGE_VERSION}
 DEPLOYMENT_ES_HEAP=${DEPLOYMENT_ES_HEAP}
 DEPLOYMENT_ONLYOFFICE_HTTP_PORT=${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}
-EXO_HOST_PORT=${EXO_HOST_PORT}
+ONLY_OFFICE_HOST_PORT=${ONLY_OFFICE_HOST_PORT}
```
Comment (vsellier) [inline etc/adt/config.template]: Sorry about that, but I just  realized the variable should be named DEPLOYMENT_ONLY_OFFICE_HOST_PORT to be coherent with the usual naming

Comment (madhifallah) [reply etc/adt/config.template]: sounds consistent, updated in code

**Pull request merged**